swagger.json

```diff
@@ -0,0 +1,278 @@
+{
```

Comment:
I'm not sure where you got this file, but it seems like a duplicate of the other swagger.json file.